packages/ckeditor5-dev-build-tools/src/plugins/splitCss.ts

```diff
@@ -297,16 +170,21 @@ async function unifyFileContentOutput( content: string = '', minimize: boolean )
 }
 
 /**
- * Wraps `declarations` list into passed `selector`;
+ * Returns normalized stylesheet content.
  */
-function wrapDefinitionsIntoSelector( selector: string, definitions: string ): string {
-	// When definition contains `data:image` it should be in one line following the specification.
-	// Currently used tool responsible for parsing definitions tries to split each definition into new line.
-	// When `data:image` contains `SVG` with style attribute which contains CSS definitions it splits it into new lines,
-	// which breaks the CSS.
-	if ( definitions.includes( 'data:image' ) ) {
-		definitions = removeNewline( definitions );
-	}
+async function normalizeStylesheet( content: string ): Promise<string> {
```

Comment:
The name and description of this function don't tell me much about what it does.

Reply:
I've change the description of the function to show exactly whit it does.